sys-auth/pam_u2f/pam_u2f-1.1.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools flag-o-matic pam
+
+DESCRIPTION="PAM module for FIDO2 and U2F keys"
+HOMEPAGE="https://github.com/Yubico/pam-u2f"
+SRC_URI="https://developers.yubico.com/${PN/_/-}/Releases/${P}.tar.gz"
+
+LICENSE="BSD"
```

Comment:
Per https://github.com/Yubico/pam-u2f/compare/pam_u2f-1.0.8...pam_u2f-1.1.0#diff-44e0558d931cff7fd8ca7724ad817ffd7c953ec032ba572d1ff1f244656ff4ccR6, they've added a source file with the `ISC` license, so we should update `LICENSE` to reflect that.